specs/phase0/validator.md

```diff
@@ -479,6 +482,14 @@ Where
 * `aggregate` is the `aggregate_attestation` constructed in the previous section.
 * `selection_proof` is the signature of the slot (`get_slot_signature()`).
 
+##### `SignedAggregateAndProof`
+
+```python
+class SignedAggregateAndProof(Container):
+    message: AggregateAndProof
+    signature: BLSSignature
```

Comment:
IMO We need to describe how we sign the message properly. To follow the pattern of "SignedContainer", do we need a new domain `DOMAIN_AGGREGATE_AND_PROOF` or just reuse `DOMAIN_BEACON_ATTESTER` as `get_slot_signature`? I'm not sure if it's valid to reuse the domain for the different types of signatures.

Reply:
okay added new domains and explicit signing instructions!